packages/core/src/types.ts

```diff
@@ -50,6 +50,9 @@ export interface LottieViewProps {
    * represents the normalized progress of the animation. If you update this prop, the
    * animation will correspondingly update to the frame at that progress value. This
    * prop is not required if you are using the imperative API.
+   *
+   * To use the `Animated` value, the component needs to be wrapped using
+   * `Animated.createAnimatedComponent(LottieView)`
```

Comment:
This is not really needed as you can wrap any component to in order to make the props animated. It's not just specific to lottie's library.